### ISSUE TITLE
Slider input has no accessible name

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-slider#189]
### Subject of the issue
Client feedback has reported the following: 
The accessible name of the slider itself must be set (referring to the `<input type="range">`). See in the screenshot below the name property is blank.

![empty_accessible_name_property](https://github.com/adaptlearning/adapt-contrib-slider/assets/7045330/33282080-52bf-465f-8483-b13f056d2a9c)

### Your environment
* v5.28.15 Framework but applies to latest
* v4.3.7 Slider but applies to latest

### Expected behaviour
The slider `<input type="range"`> should have an appropriate `aria-label` or `aria-labelledby` property set.

Note, same issue applies to the [confidenceSlider](https://github.com/adaptlearning/adapt-contrib-confidenceSlider/issues/99).

**Comment by kirsty-hames** [comment]
In addition, [Mozilla ARIA slider role best practice](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role#best_practices) states _HTML's `<input type="range">` implicitly has the `role` of `slider`. Do not use `aria-valuemax` or `aria-valuemin` attributes on `<input type="range">` elements; use `min` and `max` instead. Otherwise, any global `aria-*` attributes and any other `aria-*` attributes applicable to the slider role._

Slider already uses `min` and `max` properties but maybe we should remove `role="slider"`, `aria-valuemax` and `aria-valuemin` as suggested?
